src/components/Root/Root.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  HashRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { App } from '../../App';
+import { HomePage } from '../../pages/HomePage';
+import { PeoplePage } from '../../pages/PeoplePage';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
```

Comment:
You can add all paths in the enum and use them in all places in your application.